frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -662,6 +668,42 @@ final class BloopBspServices(
     }
   }
 
+  def jvmTestEnvironment(
+      params: bsp.JvmTestEnvironmentParams
+  ): BspEndpointResponse[bsp.JvmTestEnvironmentResult] = {
+    ifInitialized(None) { (state: State, logger: BspServerLogger) =>
+      mapToProjects(params.targets, state) match {
+        case Left(error) =>
+          logger.error(error)
+          Task.now((state, Left(JsonRpcResponse.invalidRequest(error))))
+
+        case Right(projects) =>
+          val environmentEntries = for {
+            (id, project) <- projects.toList
+            dag = state.build.getDagFor(project)
+            fullClasspath = project.fullClasspath(dag, state.client).map(_.toString)
+            environmentVariables = state.commonOptions.env.toMap
+            workingDirectory = state.commonOptions.workingDirectory
+            javaOptions = project.platform match {
```

Comment:
This should be
```diff
-             javaOptions = project.platform match {
+             javaOptions <- project.platform match {
```
So that we don't return classpaths for non-JVM projects

Reply:
Hm, I'm not sure. Please note, we have list of lists here. The return value of this endpoint has  `items: List[JvmEnvironmentItem]`, and the `JvmEnvironmentItem` contains `jvmOptions: List[String]`

Reply:
Right, what if we do it like this?
```scala
            javaOptions <- project.platform match {
              case Platform.Jvm(config, _, _) => Some(config.javaOptions.toList)
              case _ => None
            }
```

The endpoint explicitly requests the JVM environment so I'm not sure if we should return the classpath for Native/JS targets 🤔